cartographer/internal/mapping/global_trajectory_builder.h

```diff
@@ -44,8 +44,10 @@ class GlobalTrajectoryBuilder : public mapping::TrajectoryBuilderInterface {
   void AddSensorData(
       const std::string& sensor_id,
       const sensor::TimedPointCloudData& timed_point_cloud_data) override {
+    CHECK(local_trajectory_builder_)
```

Comment:
Add a comment about this in the header file, i.e. that `nullptr` is acceptable but no `TimedPointCloudData` can be added in this case?

Reply:
Done.